cellarium/cas/client.py

```diff
@@ -445,10 +448,15 @@ def __validate_cells_under_quota(self, cell_count: int) -> None:
         :param cell_count: Number of cells in the input data
         """
         user_quota = self.cas_api_service.get_user_quota()
-        if cell_count > user_quota["remaining_quota"]:
+        if cell_count > (user_quota["remaining_lifetime_quota"] or cell_count):
```

Comment:
I’m having trouble understanding the logic behind this comparison:
`if cell_count > (user_quota["remaining_lifetime_quota"] or cell_count):`

Would it make sense to simplify it to `if cell_count > user_quota["remaining_lifetime_quota"]:`?
Or is there a specific reason for including the `or cell_count `condition?

Reply:
`user_quota["remaining_lifetime_quota"]` might be `None`, so in that case, it instead checks if `cell_count > cell_count`, which it never will be.  It's basically a way of checking if we've exceeded lifetime quota only if lifetime quota actually has a value.  Although it might be more clever than clear.  I'll change it so it's more clear

Reply:
Oh got it, sorry, I didn't take into consideration it could be None. Thanks for clarifying that!